<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogShowProInfo</class>
 <widget class="QWidget" name="DialogShowProInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="Type" stdset="0">
   <string>firstWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,8">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>核  对</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="className" stdset="0">
      <string>dspiInfo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,1,1,2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>输入信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="className" stdset="0">
            <string>dspiInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>车型：</string>
           </property>
           <property name="className" stdset="0">
            <string>dspiTitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbEnVehType">
           <property name="text">
            <string>一型客</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="className" stdset="0">
            <string>dspiInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>车牌：</string>
           </property>
           <property name="className" stdset="0">
            <string>dspiTitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbEnVehPlate">
           <property name="text">
            <string>云A99999</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="className" stdset="0">
            <string>dspiInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>通行费：</string>
           </property>
           <property name="className" stdset="0">
            <string>dspiTitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbFee">
           <property name="text">
            <string>6元</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="className" stdset="0">
            <string>dspiInfo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,1,1,3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>卡内信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="className" stdset="0">
            <string>dspiInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>车型：</string>
           </property>
           <property name="className" stdset="0">
            <string>dspiTitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbVehType">
           <property name="text">
            <string>一型客</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="className" stdset="0">
            <string>dspiInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>车牌：</string>
           </property>
           <property name="className" stdset="0">
            <string>dspiTitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbVehPlate">
           <property name="text">
            <string>蓝云A99999</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="className" stdset="0">
            <string>dspiInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>卡号：</string>
              </property>
              <property name="className" stdset="0">
               <string>dspiTitle</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbCardNum">
              <property name="text">
               <string>100000000</string>
              </property>
              <property name="className" stdset="0">
               <string>dspiTitle</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>类型：</string>
              </property>
              <property name="className" stdset="0">
               <string>dspiTitle</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="lbCardType">
              <property name="text">
               <string>储值卡</string>
              </property>
              <property name="className" stdset="0">
               <string>dspiTitle</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>余额：</string>
              </property>
              <property name="className" stdset="0">
               <string>dspiTitle</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lbRest">
              <property name="text">
               <string>1500.00元</string>
              </property>
              <property name="className" stdset="0">
               <string>dspiTitle</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>widget_2</zorder>
  <zorder>line</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
